docs/crictl.md

```diff
@@ -0,0 +1,122 @@
+# Container Runtime Interface (CRI) CLI
+
+crictl provides a CLI for CRI-compatible container runtimes. This allows the CRI runtime developers to debug of their runtime without needing to set up Kubernetes components.
+
+crictl is currently Alpha and still under quick iterations. We encourage the CRI developers to report bugs or help extend the coverage by adding more functionalities.
+
+## Install
+
+The CRI CLI can be installed easily via `go get` command:
+
+```sh
+go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
+```
+
+Then `crictl` binary can be found in `$GOPATH/bin`.
+
+*Note: ensure GO is installed and GOPATH is set before installing crictl.*
+
+## Usage
+
+```sh
+crictl SUBCOMMAND [FLAGS]
+```
+
+Subcommands includes:
+
+- `info`: Get version information of runtime
+- `sandbox`, `sb`: Manage lifecycle of podsandbox
+- `container`, `ctr`: Manage lifecycle of container
+- `status`: Get the status information of runtime
+- `attach`: Attach a running container
+- `image`: Manage images
+- `exec`: Exec(exec, syncexec) a command in a running container
+- `portforward`: Forword ports(localport:remoteport) from a sandbox
+- `help`, `h`: Shows a list of commands or help for one command
+
+crictl connects to `/var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set in two ways:
+
+- By setting flags `--runtime-endpoint` and `--image-endpoint`
+- By setting environment variables `CRI_RUNTIME_ENDPOINT` and `CRI_IMAGE_ENDPOINT`
+
```

Comment:
And we also support `By setting the endpoint in the config file`, put the usage of config file here?